app/views/courses/TeacherClassView.coffee

```diff
@@ -0,0 +1,176 @@
+RootView = require 'views/core/RootView'
+template = require 'templates/courses/teacher-class-view'
+helper = require 'lib/coursesHelper'
+InviteToClassroomModal = require 'views/courses/InviteToClassroomModal'
+ActivateLicensesModal = require 'views/courses/ActivateLicensesModal'
+
+Classroom = require 'models/Classroom'
+Classrooms = require 'collections/Classrooms'
+LevelSessions = require 'collections/LevelSessions'
+Users = require 'collections/Users'
+Courses = require 'collections/Courses'
+CourseInstance = require 'models/CourseInstance'
+CourseInstances = require 'collections/CourseInstances'
+Campaigns = require 'collections/Campaigns'
+
+module.exports = class TeacherClassView extends RootView
+  id: 'teacher-class-view'
+  template: template
+  
+  events:
+    'click .add-students-button': 'onClickAddStudents'
+    'click .enroll-student-button': 'onClickEnrollStudent'
+    'click .sort-by-name': 'sortByName'
+    'click .sort-by-progress': 'sortByProgress'
+    'click #copy-url-btn': 'copyURL'
+    'click #copy-code-btn': 'copyCode'
+    'click .enroll-student-button': 'onClickEnroll'
+    'click .assign-to-selected-students': 'onClickBulkAssign'
+    'click .enroll-selected-students': 'onClickBulkEnroll'
+
+  initialize: (options, classroomID) ->
+    super(options)
+    @progressDotTemplate = require 'templates/courses/progress-dot'
+    
+    @sortAttribute = 'name'
+    @sortDirection = 1
+    
+    @classroom = new Classroom({ _id: classroomID })
+    @classroom.fetch()
+    @supermodel.trackModel(@classroom)
+    
+    @listenTo @classroom, 'sync', ->
+      @students = new Users()
+      @students.fetchForClassroom(@classroom)
+      @supermodel.trackCollection(@students)
+      @listenTo @students, 'sync', @sortByName
+      @listenTo @students, 'sort', @render
+      
+      @classroom.sessions = new LevelSessions()
+      @classroom.sessions.fetchForAllClassroomMembers(@classroom)
+      @supermodel.trackCollection(@classroom.sessions)
```

Comment:
Should only track collection if the classroom has members. Otherwise, no network requests are made, and the page never loads. Sorry for the gotcha, I think in the future, SuperModel should just be handed the list of jqxhr objects and track those.